arviz/tests/base_tests/test_stats.py

```diff
@@ -40,6 +40,23 @@ def non_centered_eight():
     return non_centered_eight
 
 
+@pytest.fixture
```

Comment:
I think we should make it have scope "module", there is no need to rerun the fixture function for every parameter in the test right?

Reply:
OK, I will change that.